tf_files/configs/kube-setup-fence.sh

```diff
@@ -43,9 +43,15 @@ if [[ -f "${WORKSPACE}/${vpc_name}_output/creds.json" ]]; then # update secrets
   mkdir -p jwt-keys
   mkdir -p ssh-keys
 
-  if [ ! -f jwt-keys/jwt_public_key.pem ]; then
-    openssl genrsa -out jwt-keys/jwt_private_key.pem 2048
-    openssl rsa -in jwt-keys/jwt_private_key.pem -pubout -out jwt-keys/jwt_public_key.pem
+  # Create keypairs for fence. Following the requirements from fence, the
+  # keypairs go in subdirectories of the base keys directory, where the
+  # subdirectories are named as an ISO 8601 timestamp of when the keypair is
+  # created.
+  if [ ! -f jwt-keys/*/jwt_public_key.pem ]; then
+    timestamp=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+    mkdir jwt-keys/${timestamp}
+    openssl genrsa -out jwt-keys/${timestamp}/jwt_private_key.pem 2048
+    openssl rsa -in jwt-keys/${timestamp}/jwt_private_key.pem -pubout -out jwt-keys/jwt_public_key.pem
```

Comment:
This code has moved to kube-setup-secrest in master branch - it will probably be easier to just create a new clean branch, and add this new login to kube-setup-secrets

Reply:
Merged master into this branch in 8e16208, moved this into `kube-setup-secrets.sh`